### ISSUE TITLE
[BUG] Coke Lab Raw Coca Table is broken.

### ISSUE THREAD

**Issue by MrMeloff** [ParadoxGamingRP/bugs-and-suggestions#1958]
### Meloff

Bug when curing coke leaves

### A clear and concise description of what the bug is

When processing the first 10 plants at the table in the coke lab, it only yields 12 cured leaves for 10 plants at 100% health.  This has happened at several labs over the last week and I am absolutely tired of wasting my time to get almost nothing after 3 hours...  This is absurd and needs to be fixed immediately.

### Steps to Reproduce the Bug

Place 10 Soaked Cocaine leaves at 100% health into the Raw Coca Processing table in any of the coke labs in the city.
Process the plants successfully completing the reflex check (hitting E).
Yields only 12x Cured Coca Leaves.
Next 10 yields 120 as normal.

### Expected Behavior

At 100%, each Soaked Coca Leaves should yield 12 leaves, not 1.2 leaves each as the bug causes...  This is ridiculous and has happened at least 3-4 times now in the last week + since the public lab was re-opened.  I refuse to waste anymore time.

### Screenshots/Videos

![image](https://github.com/ParadoxGamingRP/bugs-and-suggestions/assets/125165794/f6bad512-1d59-4222-a19a-dceb6e9119f3)

### Extra Notes (Optional)

The image linked shows the remaining 10 plants I haven't processed along with the 12 cured leaves I was given for the 1st 10 plants I tried processing.  The other 10 at the top I came with the finish the paste I had leftover.  Please fix this, I cannot waste 3 hours of my time for absolutely fuck all.

**Comment by xXJamieXx** [comment]
Unable to reproduce this...

https://github.com/ParadoxGamingRP/bugs-and-suggestions/assets/11248045/ebb2a50d-4754-4b13-a8c0-be5cdfe4ffe7

**Comment by MrMeloff** [comment]
I do apologize, I asked my other two friends who had this issue and it has come down to some extra details.  It seems to happen at 2021 and if you fail the first skill check and cancel during the slower processing bar and then try again, it seems to only read 1-3 of the 10 plants in the box.  I hope this helps, sorry I didn't have this on my first submission.

**Comment by xXJamieXx** [comment]
@MrMeloff I appreciate the fast response, unfortunately I have tried again under these conditions and am still unable to reproduce the issue. If you could possible provide a video recording of the bug that would be appreciated.

**Comment by MrMeloff** [comment]
I will do my best to get one asap.  Sadly I'm not sure what else may have caused it.

**Comment by MrMeloff** [comment]
@xXJamieXx The only thing I can think of to add to this for a future attempt would be, start with 20 plants in your inventory and then try failing on the fastest E reflex check (passing the 1st slower one(s)) and then try to process again right away.  Maybe even try failing a few times in a row this way.  Otherwise, I can't think of much else that we did that would have caused this issue.  Thanks for trying, I've always appreciated you guys and how quickly you address issues,. Sorry for my original snappy tone when I wrote the first post.
